### DISCUSSION TITLE
Detail detail details....

### DISCUSSION THREAD

**Discussion by vascolp** [pybricks/support#1188] [Q&A]
Today its just minor details....

In pybricksdev (1.0.0-alpha.46), in the method `pybricksdev.ble.BLEConnection.connect()`, and since this is also a library, shouldn't those `print()`s be `logger.debug()`s?

Another thing I noticed some time ago: pybricks `uselect.Poll.register` has in its comments: 
`register(object, eventmask=POLLOUT | POLLOUT)`.
Should be `POLLIN|POLLOUT`, I suppose...

**Comment by dlech** [comment]
> shouldn't those `print()`s be `logger.debug()`s?

Happy to take a pull request to change it to `logger.info()` (or `logger.debug()` as long as we add prints somewhere else when `pybricksdev` is used as a command line tool).



> `register(object, eventmask=POLLOUT | POLLOUT)`.

Looks like this is in a doc comment. Happy to take a pull request to fix it as well.

https://github.com/pybricks/pybricks-api/blob/821a3e44555199c4d6b5da6803accef598d7222e/src/uselect/__init__.py#L40

**Comment by vascolp** [comment]
Thank you!
And two more just found:

Code: the method `pybricksdev.ble.BLEConnection.write()` should be using `char_tx_UUID` and not `char_rx_UUID`.
It does not make a difference because `BLERequestsConnection()` sets both to the same but anyway...
Documentation: for `pybricksdev.ble.BLEConnection`: parameter list has `char_rx_UUID` twice instead of `char_rx_UUID` and `char_tx_UUID`.

**Reply by vascolp** [reply]
About the code... I was wrong, I write in RX and read in TX... it's the hub perspective.
